pkg/security/resolvers/process/resolver_linux.go

```diff
@@ -979,6 +979,19 @@ func (p *Resolver) GetProcessEnvp(pr *model.Process) ([]string, bool) {
 	return pr.Envp, pr.EnvsTruncated
 }
 
+// GetProcessEnvpScrubbed returns the scrubbed envs of the event with their values
+func (p *Resolver) GetProcessEnvpScrubbed(pr *model.Process) ([]string, bool) {
```

Comment:
I'm not sure that the scrubber works with environment variables. I would rather use the `FilterEnvs` function which removing the values expect for the allowed keys.